core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/appRoutesModels.scala

```diff
@@ -33,7 +34,8 @@ final case class CreateAppRequest(kubernetesRuntimeConfig: Option[KubernetesRunt
                                   customEnvironmentVariables: Map[String, String],
                                   descriptorPath: Option[Uri],
                                   extraArgs: List[String],
-                                  sourceWorkspaceId: Option[WorkspaceId]
+                                  sourceWorkspaceId: Option[WorkspaceId],
+                                  autoDeleteThresholdDuration: Option[Duration]
```

Comment:
Is the choice of a `scala.concurrent` library intentional? It seems like a really nice expressive contract but I wonder about this note at the top of the [doc](https://www.scala-lang.org/api/current/scala/concurrent/duration/Duration.html): 

> This class is not meant as a general purpose representation of time, it is optimized for the needs of scala.concurrent.

Would it make sense to use a `java.time` library instead?

Reply:
Actually I am not sure the pros/cons. 
Seems this class is widely used in Leo: https://github.com/search?q=repo%3ADataBiosphere%2Fleonardo%20%22scala.concurrent.duration%22&type=code. so I inherit this pattern ?

Reply:
this is intentional...a threshold is a duration..not a timestamp

Reply:
> this class is widely used in Leo
You're right - there's definitely a precedent. 👍 

> what do you mean [by an 'expressive contract']?
I meant that the Duration class looks pretty easy to use, powerful, and intuitive to understand.

The biggest thing I wanted to flag was the note in the javadoc that said it was "not meant as a general purpose representation of time" which made me think `java.time` [might be considered more appropriate for this use case](https://stackoverflow.com/questions/3614380/whats-the-standard-way-to-work-with-dates-and-times-in-scala-should-i-use-java). But a Java lib is less convenient than a Scala one!

All told, this choice is OK with me, not a blocker.